styles.css

```diff
@@ -0,0 +1,595 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Helvetica, Arial, sans-serif;
+}
+
+/* -------------------- THE NAVBAR --------------- */
+
+header {
+  position: sticky;
+  top: 0;
+  background-color: #fff;
+  z-index: 30;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5px 15px;
+  border-bottom: 1px solid #e2e2e2;
+}
+
+.nyt-logo {
+  width: 195px;
+}
+
+.left {
+  color: black;
+}
+
+.left * {
+  margin: 0 5px;
+}
+
+.left-div a {
+  text-decoration: none;
+}
+
+.left a {
+  text-decoration: none;
+}
+
+.flex-item a {
+  text-decoration: none;
+  color: #121212;
+  font-family: georgia, "times new roman", times, serif;
+}
+
+.bottom-of-article a {
+  color: rgb(102, 102, 102);
+}
+
+.last-flex-container a {
+  font-size: 14px;
+  text-decoration: none;
+  padding-top: 5px;
+  line-height: 25px;
+}
+
+.picks a {
+  display: flex;
+  position: absolute;
+  text-decoration: none;
+  font-family: Georgia, "Times New Roman", Times, serif;
+}
+
+.author a {
+  color: #000;
+  text-decoration: underline;
+  text-decoration-style: dotted;
+  font-weight: bold;
+}
+
+.right-div a {
+  font-family: Georgia, "Times New Roman", Times, serif;
+  font-size: 17px;
+  line-height: 23px;
+  color: #000;
+  text-decoration: none;
+}
+
+.left a:visited {
+  color: black;
+}
+
+.right button {
+  margin: 0 5px;
+  background-color: #567b95;
+  color: #fff;
+  padding: 11px 12px 8px 12px;
+  border: none;
+  border-radius: 3px;
+  font-weight: 600;
+  text-transform: uppercase;
+  font-size: 11px;
+}
+
+/* -------------------- Main Section --------------- */
+
+.article-title {
+  width: 600px;
+  margin: 50px auto 15px auto;
+}
+
+.main-image {
+  max-width: 1200px;
+  width: 100%;
+  margin: auto;
+}
+
+.main-image img {
+  width: 100%;
```

Comment:
- [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)